lib/requirements_config.yaml

```diff
@@ -0,0 +1,12 @@
+test_kit_id: bulk-data-test-kit
+
+suites: 
+  - id: bulk_data_test_kit
```

Comment:
There are two suites defined with the following ids:
- [bulk_data_v101](https://github.com/inferno-framework/bulk-data-test-kit/blob/main/lib/bulk_data_test_kit/v1.0.1/bulk_data_test_suite.rb)
- [bulk_data_v200](https://github.com/inferno-framework/bulk-data-test-kit/blob/main/lib/bulk_data_test_kit/v2.0.0/bulk_data_test_suite.rb)

You have the test suite for the 2.0.0 version represented, but need to update the Id. Consider adding an entry for the 1.0.1 version even if we aren't planning to test it at this time.